ports/cosmiccleaning/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "cosmiccleaning.zip",
+  "items": [
+    "Cosmic Cleaning Crew.sh",
+    "cosmiccleaning"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Cosmic Cleaning Crew",
+    "porter": [
+      "Ganimoth"
+    ],
+    "desc": "3000 AD: Earth is polluted to the point of being uninhabitable. Humans have constructed rockets to convey the populace to Mars. Catastrophe! The heavens have been crammed with trash, barring all escape. It is up to you, the Galactic Garbage Man, to save all humanity! This game was created for the GBJam 11.",
+    "desc_md": null,
+    "inst": "Ready to run.",
+    "inst_md": null,
+    "genres": [
+      "puzzle"
+    ],
+    "image": null,
+    "rtr": true,
+    "exp": false,
+    "runtime": "frt_4.0.4.squashfs",
+    "reqs": [],
+    "arch": [
+        "aarch64"
```

Comment:
runtimes do not need arch tag
```suggestion

```

Reply:
I though because of the PR [734](https://github.com/PortsMaster/PortMaster-New/pull/734/files#diff-9f25ac9de0db365a4006b3f52807239017348bfabe4ee62d277766b07ad8f345) 😁

Reply:
> I though because of the PR [734](https://github.com/PortsMaster/PortMaster-New/pull/734/files#diff-9f25ac9de0db365a4006b3f52807239017348bfabe4ee62d277766b07ad8f345) 😁

Fixed that aswell.